examples/chef/create_docker.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import optparse
+import tarfile
+import shutil
+import re
+import docker
+
+client = docker.from_env()
+
+parser = optparse.OptionParser()
+parser.add_option("-c", "--commit_sha", type='string',
+                  action='store',
+                  dest="commit_sha")
+parser.add_option("-s", "--short_sha", type='string',
+                  action='store',
+                  dest="short_sha")
+parser.add_option("-r", "--revision_id", type='string',
+                  action='store',
+                  dest="revision_id")
+parser.add_option("-b", "--build_id", type='string',
+                  action='store',
+                  dest="build_id")
+parser.add_option("-i", "--image_name", type='string',
+                  action='store',
+                  dest="image_name")
+parser.add_option("-w", "--working_dir", type='string',
+                  action='store',
+                  dest="working_dir")
+parser.add_option("-t", "--tar_path", type='string',
+                  action='store',
+                  dest="tar_path")
+parser.add_option("-p", "--platform", type='string',
+                  action='store',
+                  dest="platform")
+
+options, _ = parser.parse_args(sys.argv[1:])
```

Comment:
should we check for arguments bein empty in case someone makes a typo? name `_` to something else and validate they are empty

Reply:
Thanks for the suggestion. I added a check for each of the arguments.